mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowProtobufMapper.java

```diff
@@ -58,6 +60,21 @@ String makeSetTag(String runUuid, String key, String value) {
     return print(builder);
   }
 
+  String makeLogBatch(String runUuid, List<Metric> metrics, List<Param> params, List<RunTag> tags) {
```

Comment:
One thing I was kinda wondering (but I think we're good on) is if it'd make sense to accept a more general type here (e.g. like java.lang.Iterable<Metric>) since that's what the proto does. However we could always generalize the API further to accept Iterable instead of List in the future.

Reply:
Lemme try that ...